ModuleChangelog.md

```diff
@@ -1,10 +1,12 @@
-## 2.1.2
+## 2.2.0
 
-Release Date: Feb 15, 2023
+Release Date: February 21, 2023
 
 #### RELEASE NOTES
 
 ```
+This version of the JumpCloud ADMU will inovke unbind step as SYSTEM if running as administrator. This change should only address a limitation with administrator credentials and leaving AzureAD Domains.
```

Comment:
minor correction, invoke is misspelled. :)

Reply:
fixed :)